package/qtforkawesome/package

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(libqtforkawesome libqtforkawesome-dev)
+pkgdesc="Library that bundles ForkAwesome for use within Qt applications"
+url=https://github.com/Martchus/qtforkawesome
+pkgver=0.3.1-1
+timestamp=2025-09-26T16:18Z
```

Comment:
```suggestion
timestamp=2025-02-04T10:14Z
```
This timestamp matches the v0.3.1 release tag.

https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#timestamp-field